modules/core/src/main/java/org/apache/ignite/internal/processors/metastorage/persistence/DmsDataWriterWorker.java

```diff
@@ -81,134 +90,125 @@ public DmsDataWriterWorker(
         super(igniteInstanceName, "dms-writer", log);
         this.lock = lock;
         this.errorHnd = errorHnd;
+
+        pauseTask = new FutureTask<>(() -> AWAIT);
+        // Completed future task.
+        pauseTask.run();
```

Comment:
Instead of doing .run() you can define pauseTask as
```
private volatile Future<?> pauseTask = CompletableFuture.completedFuture(AWAIT);
```
and cast it in ``pause`` method to ``RunnableFuture`` like this
```
updateQueue.offer((RunnableFuture<?>)(pauseTask = new FutureTask<>(() -> AWAIT)));
```

Reply:
Fixed.